### DISCUSSION TITLE
Fast refresh fails in Google Chrome.

### DISCUSSION THREAD

**Discussion by steveruizok** [vercel/next.js#30126] [Help] [answered]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

16.9.1 (also fails with 14.18.1)

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

n/a

### Describe the Bug

The HMR (or fast refresh?) is freezing and failing in Chrome with the following error:

```js
workbox-7288c796.js:1 Uncaught (in promise) no-response: no-response :: [{"url":"http://localhost:3000/_next/webpack-hmr?page=whatever"}]
    at Object.D (http://localhost:3000/workbox-7288c796.js:1:22102)
    at async Object.v (http://localhost:3000/workbox-7288c796.js:1:7595)
```

The tab appears to freeze and be unable to load, even on refresh.

If I kill the dev server, I get the following errors:

![image](https://user-images.githubusercontent.com/23072548/138076230-23e47f37-85f3-4734-a06c-3fab1c7bd9c7.png)

![image](https://user-images.githubusercontent.com/23072548/138076263-07352977-848b-4167-8c21-a88bbdab0140.png)

This fails on a new project (e.g. from `yarn create next-app`) as well as older projects, back to `next: 9.5.2`, which makes me wonder if it isn't an issue in Chrome. I have not encountered this bug in Safari.

### Expected Behavior

I would expect HMR / fast refresh to work as normal in Chrome.

### To Reproduce

- Create a new Next.js project
- Yarn dev
- Make changes that would prompt a refresh

**Comment by ijjk** [comment, marked as answer]
Hi, this looks like a cache issue with a configured `service-worker`, see [here](https://stackoverflow.com/questions/33704791/how-do-i-uninstall-a-service-worker) for how to clear the service worker
